es7/2017-01/jan-24.md

```diff
@@ -0,0 +1,1449 @@
+# January 24, 2017 Meeting Notes
+
+Allen Wirfs-Brock (AWB), Waldemar Horwat (WH), Jordan Harband (JHD), Brian Terlson (BT), Michael Ficarra (MF), Adam Klein (AK), Chip Morningstar (CM), Dave Herman (DH),  Kent C. Dodds (KCD), Kevin Gibbons (KG), Tim Disney (TD), Daniel Ehrenberg (DE), Shu-yu Guo (SYG), Michael Saboff (MS), James Kyle (JK), Franziska Hinkelmann (FH), Anna Henningsen (AH), John Lenz (JL), Sebastian Markbage (SM), Bradley Farias (BF), Jeff Morrison (JM), Tyler Kellen (TK), Gabriel Isenberg (GI), James Snell (JSL), Maggie Pint (MPT), Chris Hyle (CH), Gabriel Isenberg (GI), Bert Belder (BB), Zibi Braniecki (ZB), Jamund Ferguson (JXF), Mathias Bynens (MB), Leo Balter (LB)
+
+## Adopting the agenda
+
+- Start with Istvan's items to get them out of the way
+- Jeff Morrison more free on Tuesday, preferably
+AK: This may take less than three days.
```

Comment:
Needs space before this line, otherwise runs together: 

![](https://i.gyazo.com/83b11f6a115447c738cc716dd3eda4ff.png)

All notes should be reviewed to ensure there is a blank line between statements by different attendees.